backend/migrations/002_fix_duplicate_emails.py

```
@@ -0,0 +1,271 @@
+    logger.info("AUTO-FIX MODE: Keeping oldest account for each email")
+    logger.info("=" * 60)
+    logger.info("")
+
+    total_deleted = 0
+
+    for dup in duplicates:
+        email = dup["_id"]
+        users = dup["users"]
+
+        # Sort by created_at (oldest first)
+        sorted_users = sorted(users, key=lambda x: x.get("created_at", datetime.min))
```

Ruff (0.14.7)

Use of `datetime.datetime.min` without timezone information

(DTZ901)